File: OP_PROJECT_PYTHON/src/game/Plants/SowThistle.py
from src.game.Plant import Plant


class SowThistle(Plant):
    SOW_THISTLE_STRENGTH = 0
    SOW_THISTLE_TYPE = "Sow Thistle"

    def __init__(self, position, world):
        super().__init__(SowThistle.SOW_THISTLE_TYPE, SowThistle.SOW_THISTLE_STRENGTH, position, world)

    def action(self):
        for _ in range(3):
            super().action()

File: OP_PROJECT_JAVA/src/game/Plants/SowThistle.java
package game.Plants;

import game.Plant;
import game.Position;
import game.World;
public class SowThistle extends Plant {

    private static final int SOW_THISTLE_STRENGTH = 0;
    private static final String SOW_THISTLE_TYPE = "Sow Thistle";

    public SowThistle(Position position, World world) {
        super(SOW_THISTLE_TYPE, SOW_THISTLE_STRENGTH, position, world);
    }

    @Override
    public void action() {
        for (int i = 0; i < 3; ++i) {
            super.action();
        }
    }
}

